pages/fellows/peterridolfi.md

```diff
@@ -15,9 +15,7 @@ e-mail: pbr11@pitt.edu
 project_title: Designing and implementing a converting tool for statistical models between pyhf and CMS combine
 project_goal: >
     The main goal of this project is to develop a tool that actively identifies and accommodates similarities and nuances between pyhf and CMS Combine models. The tool must be able to convert the given parameters form one type of model to the other type, while maintaining a high degree of accuracy between the actual functionality and predictions that are made by the models. Additionally, the tool should be able to give feedback concerning expected differences in these outputs based on the parameters that are given, and the limitations of the tool itself. Ultimately, the tool will serve as another method for better understanding the models that are created and the predictions that they make.
-    <br>
-    GitHub url: <a href="https://github.com/peterridolfi/Pyhf-to-Combine-converter.git">https://github.com/peterridolfi/Pyhf-to-Combine-converter.git</a>
-focus-area: as
+git-hub url: https://github.com/peterridolfi/Pyhf-to-Combine-converter.git
```

Comment:
```suggestion
```

Hm. @henryiii @rct225 from https://iris-hep.org/docs/fellow it doesn't look like there is any schema field for a project URL. Do we not support this?

Reply:
@matthewfeickert We do have github-username, which we present as their GitHub repo on their page.  I suppose we can add a field to the project for project-urls.  I prefer "project-url" since some people are using gitlab as well as GitHub.

Reply:
Okay well sounds like it isn't something that is currently supported in the spec so I'll make an Issue from this and close this.

Reply:
That would be best for now.  The "url" under the presentation section is generally for the slides used during their talk, similar to what we do for IRIS-HEP member presentations and talks.